contracts/oracles/ChainlinkOracle.sol

```diff
@@ -14,11 +14,11 @@ contract ChainlinkOracle is IOracle{
      * decShift: 8
      */
     constructor(address oracle_, uint shift) public {
-        oracle = oracle_;
+        oracle = AggregatorInterface(oracle_);
         decimalShift = shift;
     }
 
     function latestPrice() external override view returns (uint) {
-        return uint(AggregatorInterface(oracle).latestAnswer());
+        return uint(oracle.latestAnswer());
```

Comment:
`latestAnswer` from `AggregatorInterface` is actually deprecated. For closer to real-time updates moving forward, we should use `latestRoundData` from `AggregatorV3Interface`.